src/OpenDebugAD7/AD7DebugSession.cs

```diff
@@ -805,16 +816,29 @@ protected override void HandleAttachRequestAsync(IRequestResponder<AttachArgumen
         {
             const string telemetryEventName = DebuggerTelemetry.TelemetryAttachEventName;
 
+            Dictionary<string, JToken> caseInsensitiveAttachConfiguration;
+            try
+            {
+                caseInsensitiveAttachConfiguration = new Dictionary<string, JToken>(responder.Arguments.ConfigurationProperties, StringComparer.OrdinalIgnoreCase);
+            }
+            catch (ArgumentException e)
+            {
+                responder.SetError(CreateProtocolExceptionAndLogTelemetry(telemetryEventName, 1013, "Duplicate keys found in attach configuration."));
```

Comment:
Same message as above. the format is `attach: duplicate keys found in configuration`

Reply:
True, I incorrectly read how MIEngine handled JSON parsing. It looks like it also explicitally looks for `MIMode`.
https://github.com/microsoft/MIEngine/blob/403592c998c736fa75fa7aa6cd1deca259d8837a/src/MICore/JsonLaunchOptions.cs#L54